tools/interop_matrix/client_matrix.py

```diff
@@ -301,6 +301,7 @@ def __init__(self, patch=[], runtimes=[], testcases_file=None):
             ("v1.61.2", ReleaseInfo(runtimes=["go1.19"])),
             ("v1.62.2", ReleaseInfo(runtimes=["go1.19"])),
             ("v1.63.2", ReleaseInfo(runtimes=["go1.19"])),
+            ("v1.64.0", ReleaseInfo(runtimes=["go1.x"])),
```

Comment:
I see that Java and CPP does not specify a `runtimes` at all. Should we consider that option?

Reply:
Java and CPP are using the default runtimes defined above. I've done the same for Go now.